tests/conftest.py

```diff
@@ -86,6 +74,89 @@
     reason="""Requires DPF.""",
 )
 
+
+def has_dependency(requirement):
+    try:
+        import_module(requirement)
+        return True
+    except ModuleNotFoundError:
+        return False
+
+
+def requires(requirement: str):
+    """Check requirements"""
+    requirement = requirement.lower()
+
+    if "grpc" == requirement:
+        return skip_if_no_has_grpc
+
+    elif "dpf" == requirement:
+        return skip_if_no_has_dpf
+
+    elif "local" == requirement:
+        return skip_if_not_local
+
+    elif "nocicd" == requirement:
+        return skip_if_on_cicd
+
+    elif "xserver" == requirement:
+        return skip_no_xserver
+
+    elif "nolinux" == requirement:
+        return skip_on_linux
+
+    elif "nowindows" == requirement:
```

Comment:
skip and no are kind of double negatives.. kind of hard to parse.. but I assume the logic is correct

Reply:
that will look like:

```py
@requires("nolinux")
def blabla...: #Skiped if on linux.

```

Reply:
I guess I could change that.

Reply:
@germa89 up to you.. just giving my reaction

Reply:
No. you are right. I just changed: https://github.com/ansys/pymapdl/pull/2443/commits/a28898aab4c68293f890e8bf2a225da6d9c34eec